Paintroid/src/main/java/org/catrobat/paintroid/tools/drawable/HeartDrawable.kt

```diff
@@ -23,6 +23,14 @@ import android.graphics.Paint
 import android.graphics.Path
 import android.graphics.RectF
 
+private const val CONSTANT_1 = 8f
```

Comment:
hm unfortunately I also don't know the meaning of all these values, maybe somebody else knows?

Reply:
There are a few algorithms to create standard shapes like hearts. These constants are normally values like, length of a line, center position and so on. I provided @CodeChamp-SS with a SO-problem where a few of those are discussed.